12-kubernetes-cluster.tf

```diff
@@ -35,8 +35,8 @@ resource "azurerm_kubernetes_cluster" "kubernetes_cluster" {
 
   sku_tier = var.sku_tier
   default_node_pool {
-    name                 = var.project == "ss" ? "system" : "nodepool"
-    node_taints          = var.project == "ss" ? ["CriticalAddonsOnly=true:NoSchedule"] : []
+    name                 = var.enable_user_system_nodepool_split == "true" ? "system" : "nodepool"
```

Comment:
in the other PR you set this to be a boolean does this string true work properly?

Reply:
oops meant to be a bool